src/styles/index.css

```diff
@@ -0,0 +1,120 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__title {
+  display: flex;
+  font-weight: 500;
+  font-size: 12px;
+  margin-top: 40px;
+  margin-bottom: 0;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__info {
+  width: 98px;
+  height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  margin-right: 68px;
+  margin-top: 4px;
+  margin-bottom: 16px;
+  line-height: 14px;
+  color: #616070;
+}
+
+img {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+  display: block;
```

Comment:
The 'display: block;' property is declared twice for the 'img' selector. You can remove the redundant declaration.